CRM/Contact/BAO/Group.php

```diff
@@ -384,6 +384,22 @@ public static function &create(&$params) {
       $params['modified_id'] = $cid;
     }
 
+    // CRM-19068.
+    // Validate parents parameter when creating group.
+    if (isset($params['parents'])) {
+      if (is_array($params['parents'])) {
+        foreach ($params['parents'] as $parent => $dc) {
+          if (!CRM_Utils_Type::validate($parent, 'Integer', FALSE)) {
+            unset($params['parents'][$parent]);
```

Comment:
Just curious... in the case where the parameter is malformed, is it better to throw an error or ignore the parameter?

As a rule of thumb, I favor errors (because we can't follow through on the caller's intended action), but I don't know this particular code well, and maybe there's a reason why ignoring is better in this case?